security/openconnect/src/opnsense/mvc/app/models/OPNsense/Openconnect/General.xml

```diff
@@ -37,14 +37,43 @@
                     <sha1>SHA1</sha1>
                 </OptionValues>
         </hash>
-        <group type="TextField">
+        <compression type="OptionField">
+            <multiple>N</multiple>
+            <Required>N</Required>
+                <OptionValues>
+                    <none>No Compression</none>
+                    <stateless>Stateless</stateless>
+                    <all>Stateful</all>
+                </OptionValues>
+        </compression>
+        <authgroup type="TextField">
             <Required>N</Required>
             <mask>/^[a-zA-Z0-9]{1,64}$/</mask>
             <ValidationMessage>Only a-zA-Z0-9 are allowed.</ValidationMessage>
-        </group>
+        </authgroup>
+        <usergroup type="TextField">
+            <Required>N</Required>
+            <mask>/^[a-zA-Z0-9]{1,64}$/</mask>
+            <ValidationMessage>Only a-zA-Z0-9 are allowed.</ValidationMessage>
```

Comment:
This text is the error messages displayed to the user when the validation fails and should be more understandable by humans. Also the length restriction is missing